### ISSUE TITLE
Accept a `name` setting for the session name

### ISSUE THREAD

**Issue by brianp** [brianp/muxed#26]
Currently the session name is based off the filename of the project file.
Allow a `name` option in the `settings.yml` to define a session name.

The name should not be escaped, downcased, underscored, or otherwise manipulated.

ex.
`muxed.yml`

```ruby
name: "This Project"
root: "~/"
windows:
  - editor:
      layout: "main-vertical"
      panes: ["vi", "ls -alh"]
  - processes: "ls /proc"
  - logs: "tail -f /var/log/dmesg"
```

When a tmux session is created and checked using `tmux ls` the result will be:

Before:
```shell
muxed: 3 windows (created Sun Sep 25 21:23:04 2016) [123x29]
```

After:
```shell
This Project: 3 windows (created Sun Sep 25 21:23:04 2016) [123x29]
```

**Comment by brianp** [comment]
Additional expected behaviour:  

Given a project file `~/.muxed/garnish.yml`

With the contents:
```yaml
name: "Garnish Project"
windows:
  - logs: "tail -f /var/log/dmesg"
```

---

- When muxed is initialized with `$ muxed garnish`
- A session with name "Garnish Project" should be created.

---

- When the session has already been created, and `$ muxed garnish` is called
- The existing session "Garnish Project" should be attached too.

**Comment by brianp** [comment]
Fixed with #27
